components/Deck/ContentPanel/util/ChartRender.js

```diff
@@ -0,0 +1,100 @@
+class ChartRender {
+
+    static renderCharts(resize){
+
+        let charts = $('div[id^=chart]');
+
+        if (charts.length === 0 ) {
+            return 0;
+        }
+
+        for (let i = 0; i < charts.length; i++){
+
```

Comment:
To make the code more readable, I think we should not have several empty lines in a row. They are also at the beginning of the block, which makes not much sense to me.